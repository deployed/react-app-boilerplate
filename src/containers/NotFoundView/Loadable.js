/**
 * Asynchronously loads the component for NotFoundView
 */
import Loadable from 'react-loadable';


export default Loadable({
  loader: () => import('containers/NotFoundView'/* webpackChunkName: "NotFoundView" */),
  loading: () => null,
});
