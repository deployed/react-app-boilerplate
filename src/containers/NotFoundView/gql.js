import gql from 'graphql-tag';


export const redirectUrlQuery = gql`
  query RedirectUrl($url: String){
    redirectUrl(url: $url){
      url
    }
  }
`;
