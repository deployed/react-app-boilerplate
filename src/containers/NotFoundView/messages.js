/*
 * NotFoundView Messages
 *
 * This contains all the text for the NotFoundView component.
 */
import { defineMessages } from 'react-intl';


export default defineMessages({
  header: {
    id: 'app.components.NotFoundView.header',
    defaultMessage: 'Seite nicht gefunden',
  },
});
