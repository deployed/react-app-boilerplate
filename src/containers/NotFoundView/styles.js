import styled from 'styled-components';

import image from './404.png';


export const NotFoundViewImage = styled.article`
  height: 350px;
  background: rgb(58, 122, 134);
  background: url(${image}) no-repeat center center;
  background-size: cover;
  padding: 25px;
  text-shadow: 2px 4px 3px rgba(0, 0, 0, 0.3);
`;
