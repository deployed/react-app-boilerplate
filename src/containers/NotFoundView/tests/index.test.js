import React from 'react';
import { shallow } from 'enzyme';

import NotFoundView from '../index';


describe('<NotFoundView />', () => {
  it('match snapshot', () => {
    const renderedComponent = shallow(
      <NotFoundView />,
    );
    expect(renderedComponent).toMatchSnapshot();
  });
});
