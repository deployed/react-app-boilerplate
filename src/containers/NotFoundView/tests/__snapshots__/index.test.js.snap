// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundView /> match snapshot 1`] = `
<React.Fragment>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <meta
      content="noindex, nofollow"
      name="robots"
    />
  </HelmetWrapper>
  <div>
    404: Not Found View
  </div>
</React.Fragment>
`;
